typescript-view-swapping/main.ts

```diff
@@ -2,11 +2,15 @@ const $TabContainer = document.querySelector('.tab-container');
 const $AllTab = document.querySelectorAll('.tab');
 const $AllView = document.querySelectorAll('.view');
 
-if (!$TabContainer) throw new Error("Didn't work:(");
+if (!$TabContainer) throw new Error('The $TabContainer query did not work.');
 
 function handleClick(event: Event): void {
   const $eventTarget = event.target as HTMLDivElement;
 
+  if (!$eventTarget) {
+    throw new Error('The $eventTarget is not selecting a tab');
+  }
```

Comment:
This only checks if _not_ `$eventTarget` but if this is running, then you will always have an `$eventTarget` We should make this more specific to make sure it is one of the `tab` elements we want to use (in the instructions there's a method that will help with that!)